### PR TITLE
feat/reward donator page

### PR DESCRIPTION
**Pull request by misterfocusth**
- Created reward home page (donator) /reward
- Created reward history page (donator) /reward/history

FYI: Tomorrow I will make another page that related to rewards (/reward/history/:id) for viewing specific reward history info you can hold this pr and merge it into main when another page is ready.

### CODE REVIEW

File: src/app.html [hunk 1]
```diff
@@ -3,6 +3,12 @@
 	<head>
 		<meta charset="utf-8" />
 		<link rel="icon" href="%sveltekit.assets%/favicon.png" />
```
Comment (DrowningToast) [inline src/app.html]: import the font in the global css file instead of root HTML file

File: src/routes/(donator)/reward/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+	export let data: PageData;
+</script>
+
+<div class="bg-[#F5222D] min-h-screen">
+	<div class="bg-white shadow-md p-5 flex flex-row items-center justify-start gap-4">
+		<button
+			on:click={() => {
+				if (browser) {
+					window.history.back();
```
Comment (DrowningToast) [inline src/routes/(donator)/reward/+page.svelte]: ใช้ goto() ของ svelte ได้ไหม

File: src/routes/(donator)/reward/history/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+	export let data: PageData;
+</script>
+
+<div class="bg-[#F5F5F5] h-screen">
+	<div class="bg-white shadow-md p-5 flex flex-row items-center justify-start gap-4">
+		<button
+			on:click={() => {
+				if (browser) {
+					window.history.back();
```
Comment (DrowningToast) [inline src/routes/(donator)/reward/history/+page.svelte]: ใช้ goto() ของ svelte ได้ไหม

File: src/routes/login/+page.svelte [hunk 4]
```diff
@@ -56,12 +56,14 @@
 
 		<p class="underline text-[#F5222D] w-full text-right pr-2">ลืมรหัสผ่าน ?</p>
 
-		<Button
-			variant="secondary"
-			class="w-full rounded-xl py-6 text-md font-bold bg-[#F5222D] text-white hover:bg-red-600 active:bg-red-600"
-		>
-			เข้าสู่ระบบ
-		</Button>
+		<a href="/home">
```
Comment (DrowningToast) [inline src/routes/login/+page.svelte]: ทำไมโค้ดบางไฟล์ถึงเขียน href ข้างใน Button

**Review by DrowningToast — CHANGES_REQUESTED** [review]

**Comment by misterfocusth** [comment]
@DrowningToast all finish now you can review this pr and merge it.

**Comment by misterfocusth** [comment]
@DrowningToast there is one page left for this branch don't merge yet.

**Comment by misterfocusth** [comment]
@DrowningToast review plz.

**Review by DrowningToast — APPROVED** [review]
LGTM!

**Pull request merged**